openapi/oi.yaml

```diff
@@ -0,0 +1,1687 @@
+              schema:
+                type: string
+                default: no-store
+        "500":
+          description: Unknown error.
+          headers:
+            Pragma:
+              description: Pragma browser directive
+              schema:
+                type: string
+                default: no-cache
+            Cache-Control:
+              description: Cache-Control browser directive
+              schema:
+                type: string
+                default: no-store
+  /oidc/register:
+    options:
+      tags:
+        - Authorization Server APIs
+      operationId: Options_oidc_register
+      description: CORS preflight request
+      parameters:
+        - name: Access-Control-Request-Headers
+          in: header
+          required: true
+          description: The Access-Control-Request-Headers request header is used when issuing a preflight request to let the server know which HTTP headers will be used when the actual request is made.
+          schema:
+            type: string
+        - name: Access-Control-Request-Method
+          in: header
+          required: true
+          description: The Access-Control-Request-Method request header is used when issuing a preflight request to let the server know which HTTP method will be used when the actual request is made.
+          schema:
+            type: string
+        - name: Origin
+          in: header
+          required: true
+          description: The Origin request header indicates where a fetch originates from.
+          schema:
+            type: string
+      responses:
+        "204":
+          description: "No Content, applicable to OPTIONS request with preflight configuration"
+          headers:
+            Access-Control-Allow-Methods:
+              description: "Allowable Methods, POST"
+              schema:
+                type: string
+                default: POST
+        "400":
+          description: Bad request
+          headers:
+            Pragma:
+              description: Pragma browser directive
+              schema:
+                type: string
+                default: no-cache
+            Cache-Control:
+              description: Cache-Control browser directive
+              schema:
+                type: string
+                default: no-store
+      summary: OpenID Connect Dynamic Registration Configuration endpoint
+    post:
+      tags:
+        - Authorization Server APIs
+      description: "This endpoint follows the specification defined at http://openid.net/specs/openid-connect-registration-1_0.html. It enables a client to dynamically register itself as an OAuth 2.0 client."
+      operationId: Post_oidc_register
+      responses:
+        "201":
+          description: Successful registration including details about the registered values.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
```

Comment:
the `schema` for `oidc/register` endpoint should contain a custom parameter `spid_requested_attributes`. It could be a type `array` object that can be filled with [this set](https://docs.italia.it/italia/spid/spid-regole-tecniche/it/stabile/attributi.html) of attributes at least